packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -18,8 +18,10 @@ export default {
       detection: true,
       // Timeout duration to wait for ICE to reconnect if a disconnect is received.
       iceReconnectionTimeout: 10000,
+      // Amount of times attempting to rejoin a meeting during reconnect
+      maxRejoinAttempts: 3,
       retry: {
-        times: 2,
+        times: 5,
```

Comment:
Are we still using the backoff rate?

Reply:
Not sure why that was modified but I will remove all those configs not used.